### ISSUE TITLE
Files to generate the domain information are missing

### ISSUE THREAD

**Issue by crdanielbusch** [openmethane/openmethane-prior#7]
## The problem
Is it possible that some files are missing in the `cmaq_example` directory.  I am following the instructions in the README to generate the domain information needed for grid resolution. The script `omCreateDomainInfo.py` needs three files: `GRIDCRO2D_1`, `GRIDDOT2D_1`, and `geo_em.d01.nc`. I can see only the last one.

## Definition of "done"
It would be great if you could provide the files. Or if it works without them, a description of how to do it.

**Comment by prayner** [comment]
>The script
>`omCreateDomainInfo.py` needs three files: `GRIDCRO2D_1`,
>`GRIDDOT2D_1`, and `geo_em.d01.nc`. I can see only the last one.
That's right. I had moved to only needing the geo_em but reverted it. 
geo_em is produced by the preprocessing step for WRF. The gridcro and
griddot files are produced by the subsequent postprocessing of WRF
files to produce input meteorology for CMAQ. The catch is that the
CMAQ domain is smaller than the WRF domain, slicing some points off
the boundary to remove edge effects from the atmospheric model. The
number removed is, I think, a user option so we couldn't assume it.
Thus I moved back to the three files. This imposes a dependency of the
prior processing on the output of the CMAQ preprocessing. This is
mainly a problem now when we're setting up. All those three files are
static once a domain is set up so there's no dependency in the
production workflow but it is blocking development. My first
suggestion is that I carry through Jared's WRF tests on the test
domain to the CMAQ preprocessing stage and make the resulting griddot
and gridcro files available. Any other suggestions welcome of course.
